docs/troubleshooting.md

```diff
@@ -1,19 +1,20 @@
+| Cloud Storage FUSE not mounting after reboot when entry is present in ```/etc/fstab``` with 1 or 2 as fsck order                                                                                                                                                                        | Pass [_netdev option](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/mounting.md#persisting-a-mount) in fstab entry (reference issue [here](https://github.com/GoogleCloudPlatform/gcsfuse/issues/1043)). With this option, mount will be attempted on reboot only when network is connected.                                                                                                                                                                                                                                                                                                         |
+| Cloud Storage FUSE get stuck when using it to concurrently work with a large number of opened files (reference issue [here](https://github.com/GoogleCloudPlatform/gcsfuse/issues/1043))                                                                                                | This happens when gcsfuse is mounted with http1 client (default) and the application using gcsfuse tries to keep more than value of `--max-conns-per-host` number of files opened. You can try (a) Passing a value higher than the number of files you want to keep open to `--max-conns-per-host` flag. (b) Adding some timeout for http client connections using `--http-client-timeout` flag.                                                                                                                                                                                                                       |
+| xxx-compute@developer.gserviceaccount.com does not have storage.objects.create access to the Google Cloud Storage object. Permission 'storage.objects.create' denied on resource (or it may not exist)., forbiddenE0920 02:44:56.692508 fuse: *fuseops.MkDirOp error: permission denied | Provide roles/storage.objectAdmin role on the bucket.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Instead of this specific error, put a generic permission_denied error. And point to our cloud docs regarding what permissions are required.